src/accelerate/utils/environment.py

```diff
@@ -48,3 +50,12 @@ def parse_flag_from_env(key, default=False):
 def parse_choice_from_env(key, default="no"):
     value = os.environ.get(key, str(default))
     return value
+
+
+def are_libraries_initialized(library_names: list = []) -> Dict[str, bool]:
```

Comment:
```suggestion
def are_libraries_initialized(library_names: list[str]) -> Dict[str, bool]:
```

Alternatively, to avoid the `isinstance` check below, just do `*library_names`, then we know it's always a tuple.